dialect.loki.js

```diff
@@ -332,21 +333,23 @@ module.exports = (app, prefix) => {
     });
   });
 
-  app.get(prefix + '/loki/v1/channel/:id/get_moderators', async (req, res) => {
+  const getChannelModeratorsHandler = async (req, res) => {
     const channelId = parseInt(req.params.id);
     const roles = {
       moderators: [],
     };
-    for (const userid in user_access) {
-      try {
-        const user = await getUser(userid);
-        roles.moderators.push(user.username);
-      } catch (e) {
-        console.error(`Errer getting user: ${e}`);
-      }
-    }
+    const userids = Object.keys(user_access);
+    const userAdnObjects = await getUsers(userids);
```

Comment:
This can result in uncaught rejections with current implementation of getUsers I believe, can either resolve the error and if check or try catch?

Reply:
addressed